test/filterResults.test.ts

```diff
@@ -0,0 +1,79 @@
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import chaiString from 'chai-string'
+import fs from 'fs/promises'
+import os from 'os'
+import path from 'path'
+import spawn from 'spawn-please'
+import { FilterResultsFunction } from '../src/types/FilterResultsFunction'
+import stubNpmView from './helpers/stubNpmView'
+
+chai.should()
+chai.use(chaiAsPromised)
+chai.use(chaiString)
+
+const bin = path.join(__dirname, '../build/src/bin/cli.js')
+
+/**
+ * Sets up and tears down the temporary directories required to run each test
+ */
+async function filterResultsTestScaffold(
```

Comment:
I prefer redundancy over abstraction in tests, so let's avoid a separate test scaffold function. You can do everything inline in the test case.

Reply:
fixed 🚀